src/backend/services/context.py

```diff
@@ -28,8 +29,12 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send):
         request = Request(scope)
         context.with_deployment_name(request.headers.get("Deployment-Name", ""))
 
-        request = Request(scope)
-        user_id = get_header_user_id(request)
+        # Not all endpoints require authentication, so we need to handle the case where the user_id is not present
+        try:
+            user_id = get_header_user_id(request)
```

Comment:
Would it be better to check if authorisation is none in the func and then default? Worried this might hide another issue

Reply:
I feel like these workaround solutions will hide other issues until we handle auth in all endpoints. I can check if auth is none for now, but I'll try to come up with something better later.